plugins/all.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: all
+spec:
+  homepage: https://github.com/onatm/kall
+  shortDescription: Run 'kubectl get' on all the contexts found in your .kube/config
```

Comment:
Try to stay below 50 chars, otherwise your message will get truncated.

Reply:
Maybe `Run 'kubectl get' on contexts in your kubeconfig`?

Reply:
I'll shorten it and make it non `get` specific.